repairnator/src/main/java/fr/inria/spirals/repairnator/serializer/json/JsonSerializer.java

```diff
@@ -310,6 +332,10 @@ public void serializeData(ProjectInspector inspector) {
             case PATCHED:
                 outputHasBeenPatchedInspector(inspector);
                 break;
+                
+            case FIXERBUILD:
+            	outputFixerBuildInspector(inspector);
```

Comment:
I would like to know if you agree with a thing. For GoogleSpreadSheetInspectorSerializer, for example, can I maintain this class only for mutual things we need, and then I create two extensions of this class, one for Bears and other for RepairNator?